src/main/java/mate/academy/springboot/datajpa/services/impl/CategoryServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package mate.academy.springboot.datajpa.services.impl;
+
+import mate.academy.springboot.datajpa.models.Category;
+import mate.academy.springboot.datajpa.repositories.CategoryRepository;
+import mate.academy.springboot.datajpa.services.CategoryService;
+import org.springframework.stereotype.Service;
+
+@Service

```

Comment:
In such cases, you can use @RequiredArgsConstructor, to avoid boilerplate code with constructors.